src/main/java/core/basesyntax/figures/Circle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.figures;
+
+public class Circle extends Figures {
+    private int radius;
+
+    public Circle(int radius, String color) {
+        setColor(color);
+        this.radius = radius;
+    }
+
+    public Circle() {
+
+    }
+
+    public void setRadius(int radius) {
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.ceil((Math.PI * radius * radius) * 10) / 10;

```

Comment:
[All magic numbers in your code should be constants.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#all-magic-numbers-in-your-code-should-be-constants)
fix it for all figures